subprojects/docs/build.gradle

```diff
@@ -571,6 +572,14 @@ tasks.withType(InstallSample).configureEach {
     }
 }
 
+def isConfigurationCacheEnabledForDocsTests() {
```

Comment:
Can we move these two methods to `BuildParams`, too? There're some util methods. The kotlin method can be used in Groovy build script, too, for example [here](https://github.com/gradle/gradle/blob/293505bdd320762b04e736eae9ac1ceeaa3b9379/subprojects/docs/build.gradle#L691)